agent/discover/cluster.go

```diff
@@ -15,19 +15,25 @@ import (
 
 const crm_monPath string = "/usr/sbin/crm_mon"
 const cib_admPath string = "/usr/sbin/cibadmin"
+const ClusterDiscoverId string = "discover_cluster"
```

Comment:
Just a very dumb remark, but should we go with this syntax for declaring const or the "block" one used e.g. in https://github.com/trento-project/trento/pull/51/files#diff-c79c0069c6158bdd3d204da61ea8495ac31e344c40050f7e6e8ec298426fe998R23 ?

Reply:
Both are fine. The 2nd might be cleaner. There might be much more of those in the code. 
I would bet to use an additional PR in the future to fix them if we want to do son